### PR TITLE
Use traefik for keycloak

### PR DESCRIPTION
**Pull request by berendt**
Signed-off-by: Christian Berendt <berendt@osism.tech>

### CODE REVIEW

**Comment by berendt** [comment]
Depends-on:
- https://github.com/osism/ansible-collection-services/pull/568

File: environments/infrastructure/configuration.yml [hunk 1]
```diff
@@ -12,20 +12,21 @@ cephclient_mons:
...
 netbox_traefik: true
 
-netbox_host: "{{ hostvars[inventory_hostname]['ansible_' + internal_interface]['ipv4']['address'] }}"
-
 ##########################
 # nexus
 
 nexus_traefik: true
 
-nexus_host: "{{ hostvars[inventory_hostname]['ansible_' + internal_interface]['ipv4']['address'] }}"
```
Comment (osfrickler) [inline environments/infrastructure/configuration.yml:27]: Are these deletions intentional? Usually I would split this into a dedicated patch or at least mention it in the commit message.

File: environments/infrastructure/configuration.yml [hunk 2]
```diff
@@ -12,20 +12,21 @@ cephclient_mons:
...
-
 netbox_plugins_extra:
```
Comment (osfrickler) [inline environments/infrastructure/configuration.yml:30]: While you are cleaning up, move this into the netbox section?

File: environments/infrastructure/configuration.yml (hunk 1 above)
Comment (berendt) [reply environments/infrastructure/configuration.yml:27]: I'm moving this into a cleanup PR.

File: environments/infrastructure/configuration.yml (hunk 2 above)
Comment (berendt) [reply environments/infrastructure/configuration.yml:30]: Same.

**Comment by berendt** [comment]
Cleanups in https://github.com/osism/testbed/pull/1045
